### DISCUSSION TITLE
Bug - swapped Campaign and Crew book spots

### DISCUSSION THREAD

**Discussion by ninjew42** [bcholmes/StarTrek2d20#122] [General]
It looks like Picard S1 and Discovery Campaign swapped places
![Swapped icons](https://user-images.githubusercontent.com/121691733/212447348-a2d1d0f6-e642-4273-87bb-5369efbef786.PNG)

**Comment by bcholmes** [comment]
This should be resolved, now. Thanks for letting me know.
